### ISSUE TITLE
Many new features are rolling out. 

### ISSUE THREAD

**Issue by crypt0miester** [michaelhly/solana-py#253]
Anyone working on it? 

Need to bump docker solana version to latest. 

- New transaction [confirmation strategy](https://github.com/solana-labs/solana/blob/752c851070873a180d815f8b4e447e8b83be2a38/web3.js/src/connection.ts#L2847) using blockheights.

latest version needed for:
- [Compute Budget IX](https://github.com/solana-labs/solana/blob/master/web3.js/src/compute-budget.ts)
- Quic/http3 integration 

This also makes us ready for:
- [Prioritization Fee function](https://github.com/solana-labs/solana/issues/26017)

**Comment by crypt0miester** [comment]
alright, I'm working on it. waiting for others to check if they are doing it as well. 

the confirmation strategy is a much needed feature. it is much better than the timeouts since it relies upon block expiry rather than time based checks. 

but since the current Transaction relies on solders, I might have to look at solders Transaction as well. no idea how it works yet.

**Comment by michaelhly** [comment]
@crypt0miester I believe solders just invokes the Solana Lab's rust sdk:
https://github.com/solana-labs/solana/tree/master/sdk/src

cc @kevinheavey

**Comment by crypt0miester** [comment]
yup, I think we do nothing to the transaction, it is after it that the last valid blockheight is checked upon.

I already coded some stuff, let's see if the tests go through.

**Comment by crypt0miester** [comment]
requesting review of #254

**Comment by crypt0miester** [comment]
I already have an MVP for quic using aioquic. but the performance is not as expected. 

it loses performance on connect since it needs to look for the ip of the rpc. adds about a few hundred ms. no idea if I should continue working on it now or not. 

Thus, it is much faster to just call the rpc using http/1.1 (haven't tested it against http/2)

**Comment by whymidnight** [comment]
gm everyone,

i found a hacky solution to requesting more compute units -
```python
from solana.transaction import TransactionInstruction

            request_compute_ix = TransactionInstruction(
                keys=[],
                program_id=PublicKey("ComputeBudget111111111111111111111111111111"),
                data=bytes.fromhex("0080a9030010270000"),
            )
```

i just looked at a transaction that included the instruction and copied the instruction data as hex and reimplemented manually via `bytes.fromhex`.

**Comment by crypt0miester** [comment]
> gm everyone,
> 
> i found a hacky solution to requesting more compute units -
> 
> ```python
> from solana.transaction import TransactionInstruction
> 
>             request_compute_ix = TransactionInstruction(
>                 keys=[],
>                 program_id=PublicKey("ComputeBudget111111111111111111111111111111"),
>                 data=bytes.fromhex("0080a9030010270000"),
>             )
> ```
> 
> i just looked at a transaction that included the instruction and copied the instruction data as hex and reimplemented manually via `bytes.fromhex`.
> 
> <img alt="image" width="1003" src="https://user-images.githubusercontent.com/45319061/187131091-011a829e-b67e-4e4d-8898-9f76d3a6836e.png">

gm. how about other ixs?
